packages/core/src/router/providers/bind-element.ts

```diff
@@ -0,0 +1,15 @@
+// Copyright IBM Corp. 2017. All Rights Reserved.
+// Node module: @loopback/core
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {Binding, Context, inject, Provider} from '@loopback/context';
```

Comment:
So strangely enough, this import is needed.

Here is the compiler error I am getting otherwise:

```
packages/core/src/router/providers/bind-element.ts(12,3): error TS4053:
Return type of public method from exported class has or is using name 'Binding'
from external module
"/home/travis/build/strongloop/loopback-next/packages/context/src/binding"
but cannot be named.
```